torch/testing/_internal/common_modules.py

```diff
@@ -4110,6 +4110,9 @@ def module_error_inputs_torch_nn_Pad3d(module_info, device, dtype, requires_grad
                ),
     ModuleInfo(torch.nn.RMSNorm,
                module_inputs_func=module_inputs_torch_nn_RMSNorm,
+               skips=(
+                   # No channels_last support for RMSNorm currently.
```

Comment:
I think in practcice, layernorm/rmsnorm are used for channels_last inputs ("BTC"), right? but the logical layout matches the memory layout as is typically